test/e2e/storage/pvc_storageclass.go

```diff
@@ -74,7 +74,7 @@ var _ = utils.SIGDescribe("Retroactive StorageClass Assignment", func() {
 		// Create a PVC with nil StorageClass
 		pvc := createPVC(ctx, client, namespace)
 		ginkgo.DeferCleanup(func(cleanupContext context.Context) {
-			err := client.CoreV1().PersistentVolumeClaims(namespace).Delete(ctx, pvc.Name, *metav1.NewDeleteOptions(0))
+			err := client.CoreV1().PersistentVolumeClaims(namespace).Delete(cleanupContext, pvc.Name, *metav1.NewDeleteOptions(0))
```

Comment:
A better solution is to rename `cleanupContext` to `ctx` above and to not change this line.

Then only one context is accessible in this code and it's the one which the code is supposed to use. It's also shorter.

Reply:
Done in latest rev.